### DISCUSSION TITLE
Missing input data for Conus4v1 resolution

### DISCUSSION THREAD

**Discussion by jacob-stu-allen** [E3SM-Project/E3SM#6264] [E3SM model help]
Hello!

I'm attempting to build/submit a case with the conusx4v1_conusx4v1 resolution and an F1850 compset. I'm having no issue in building the case, but upon running case_submit, I appear to be missing these data files: 

Model elm missing file finidat = '/global/cfs/cdirs/e3sm/inputdata/lnd/clm2/initdata_map/clmi.I1850CLM45.conusx4v1.74e105b.clm2.r.0021-01-01-00000.nc'

...
Loading input file list: 'Buildconf/mpassi.input_data_list'
Model mpassi missing file mesh = '/global/cfs/cdirs/e3sm/inputdata/ice/mpas-cice/tx0.1v2/..nc'

...
Model mpassi missing file graph64 = '/global/cfs/cdirs/e3sm/inputdata/ice/mpas-cice/tx0.1v2/.part.64'

Are these locatable on Perlmutter? 

Thanks in advance,
Jacob

**Comment by jonbob** [comment]
@jacob-stu-allen -- in order to use that resolution, you'll need to use an F-compset that had CICE as the ice component and not mpassi. In order to use mpassi, the ice mesh has to be an mpas mesh -- which the conus is not. The elm finidat file is on the inputdata repo, so it should get downloaded automatically
